rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/test_is_baseline_system_10.py

```diff
@@ -53,24 +108,36 @@
 
 
 def test__TEST_RMD_baseline_system_10_is_valid():
-    schema_validation_result = schema_validate_rmr(SYS_10_TEST_RMD)
+    schema_validation_result = schema_validate_rmr(SYS_10_FIRST_LOGIC_TEST_RMD)
     assert schema_validation_result[
         "passed"
     ], f"Schema error: {schema_validation_result['error']}"
 
 
-def test__is_baseline_system_10__true():
+def test__is_baseline_system_10__first_logic_true():
     assert (
         is_baseline_system_10(
-            SYS_10_TEST_RMD["ruleset_model_instances"][0],
+            SYS_10_FIRST_LOGIC_TEST_RMD["ruleset_model_instances"][0],
             "System 10",
-            ["Air Terminal"],
+            ["Air Terminal 1"],
             ["Thermal Zone 1"],
         )
         == HVAC_SYS.SYS_10
     )
 
 
+def test__is_baseline_system_10__second_logic_true():
```

Comment:
Could you add comments to distinguish the first logic and second logic?

Reply:
Good point! Sure. I just added the comment about the first and second logic.